textattack/search_methods/metropolis_hastings.py

```diff
@@ -0,0 +1,175 @@
+import torch
+import math
+import warnings
+import numpy as np
+from textattack.shared import utils
+from textattack.search_methods import SearchMethod
+from transformers import AutoTokenizer, AutoModelWithLMHead
+
+class MetropolisHastingsSampling(SearchMethod):
+    """ 
+    Uses Metropolis-Hastings Sampling to generate adversarial samples.
+    Based off paper "Generating Fluent Adversarial Examples for Natural Langauges" by Zhang, Zhou, Miao, Li (2019)
+
+    Args:
+        max_iter (int): The maximum number of sampling to perform
```

Comment:
Can you specify in the docstring that if an input has more words than `max_iter`, the algorithm will increase the maximum iterations to the number of words? (I think this is what's happening in line 109)

Reply:
done